iseus/styles.css

```diff
@@ -180,7 +180,7 @@ section h2{
 }
 .sec2{
     background: white;
-    height: 500px;
+    height: 1525px;
```

Comment:
Please take a look at CSS units (e.g. vh, vw, rem, em). They are very handy and very common in CSS.